EMS/core-bundle/src/Controller/ContentManagement/DatatableController.php

```diff
@@ -18,8 +19,25 @@
 
 final class DatatableController extends AbstractController
 {
-    public function __construct(private readonly DatatableService $datatableService, private readonly TableRenderer $tableRenderer, private readonly TableExporter $tableExporter, private readonly TokenStorageInterface $tokenStorage)
+    public function __construct(
+        private readonly DatatableService $datatableService,
+        private readonly DataTableFactory $dataTableFactory,
+        private readonly TableRenderer $tableRenderer,
+        private readonly TableExporter $tableExporter,
+        private readonly TokenStorageInterface $tokenStorage
+    ) {
+    }
+
+    public function ajaxData(Request $request, string $cacheKey): Response
     {
+        $table = $this->dataTableFactory->createFromCache($cacheKey);
+        $dataTableRequest = DataTableRequest::fromRequest($request);
+        $table->resetIterator($dataTableRequest);
```

Comment:
Introduce a role mechanisme? It give (more or less) anonymous access to all data in my head?

Reply:
Fixed [2938003](https://github.com/ems-project/elasticms/pull/492/commits/2938003c6731170cfa81f77fcf4d68bd92a8dc10)